Varificando data de envio...
PRAZO MÁXIMO: 2021-09-27T23:59:59-03:00
Testando sua solucao...
Procurando exercício 1...
Exercício 1...
Teste 1: correto!
Teste 2: correto!
Teste 3: correto!
Teste 4: correto!
Teste 5: correto!
Teste 6: correto!
Teste 7: correto!
Teste 8: correto!
Teste 9: correto!
Teste 10: correto!
Teste 11: correto!
Teste 12: correto!
Teste 13: correto!
Teste 14: correto!
Teste 15: correto!
Teste 16: correto!
Teste 17: correto!
Teste 18: correto!
Teste 19: correto!
Teste 20: correto!
Nota exercicio 1: 1.0
Procurando exercício 2...
Exercício 2...
Teste 1: correto!
Teste 2: correto!
Teste 3: correto!
Teste 4: correto!
Teste 5: correto!
Teste 6: correto!
Teste 7: correto!
Teste 8: correto!
Teste 9: correto!
Teste 10: correto!
Teste 11: correto!
Teste 12: correto!
Teste 13: correto!
Teste 14: correto!
Teste 15: correto!
Teste 16: correto!
Teste 17: correto!
Teste 18: correto!
Teste 19: correto!
Teste 20: correto!
Nota exercicio 2: 2.0
Procurando exercício 3...
Exercício 3...
Teste 1: correto!
Teste 2: correto!
Teste 3: correto!
Teste 4 deu errado!
84
59
46
26
44
39
-41
2
-83
-50
90
21
46
-100
98
94
-16
86
-27
-68
48
-51
36
65
76
20
31
-14
-80
1
-10
-33
56
-40
-87
-26
33
-86
-18
42
9
53
96
5
42
-77
-63
19
73
2
-26
19
-34
100
-83
-84
-42
-12
-93
92
99
-19
40
89
94
-44
27
36
56
24
1
-39
18
-62
-17
-93
-65
20
40
13
44
41
-42
78
-30
100
-17
79
36
-59
-90
59
48
-85
9
-53
31
s
14
7

Traceback (most recent call last):
  File "/__w/Monitoria-PA/Monitoria-PA/lista05/ex3.py", line 15, in <module>
    m[i][j] = l[pos]
IndexError: list index out of range

Teste 5: correto!
Teste 6: correto!
Teste 7: correto!
Teste 8: correto!
Teste 9: correto!
Teste 10 deu errado!
O resultado para
83
65
-33
6
-38
23
-60
-36
5
29
80
-96
-21
-73
-35
-63
-80
22
-24
-91
59
27
-33
-31
87
-6
25
19
-29
58
-64
52
68
15
-2
-78
28
s
6
6

deveria ser
IMPOSSIVEL
mas deu
83 65 -33 6 -38 23 
-60 -36 5 29 80 -96 
-21 -73 -35 -63 -80 22 
-24 -91 59 27 -33 -31 
87 -6 25 19 -29 58 
-64 52 68 15 -2 -78 

Teste 11 deu errado!
-24
78
31
44
-69
91
2
-50
-33
-29
-90
83
-62
48
62
54
-58
-83
-61
10
-14
86
-80
-75
83
-44
62
7
37
36
63
65
-7
-9
-30
18
57
28
-65
-54
-23
-67
-38
22
-65
9
67
10
-72
7
-23
45
1
36
-46
3
-30
9
-81
-45
1
16
-44
-19
18
70
-40
-93
-30
0
7
-83
-12
-27
-74
83
16
-11
43
41
-32
-46
-73
-36
-29
-72
44
50
47
-69
10
4
-31
-97
38
79
83
-16
-11
57
2
8
14
58
-2
89
94
-45
-5
59
59
70
29
-18
-37
95
6
-15
59
8
-11
-30
74
3
-21
45
-65
45
33
63
-14
-41
3
43
-88
-67
10
-28
-46
28
2
29
26
-24
63
-34
-84
-92
42
-26
-26
-28
-22
34
-42
-73
-65
83
3
12
-53
4
-41
48
56
74
-56
95
-72
94
-54
-98
-29
-10
54
-71
-18
59
64
2
64
-48
-19
-29
98
40
-67
-21
70
21
89
-91
98
-2
60
-91
30
-2
10
43
89
25
8
-67
-31
-90
6
-72
-10
56
-75
17
25
-46
-70
-74
0
-29
42
87
4
-71
-1
83
-48
-64
-20
6
88
-30
70
37
-60
8
19
-56
43
s
17
14

Traceback (most recent call last):
  File "/__w/Monitoria-PA/Monitoria-PA/lista05/ex3.py", line 15, in <module>
    m[i][j] = l[pos]
IndexError: list index out of range

Teste 12 deu errado!
Teste 13: correto!
Teste 14 deu errado!
Teste 15 deu errado!
-69
15
3
s
1
4

Traceback (most recent call last):
  File "/__w/Monitoria-PA/Monitoria-PA/lista05/ex3.py", line 15, in <module>
    m[i][j] = l[pos]
IndexError: list index out of range

Teste 16: correto!
Teste 17: correto!
Teste 18 deu errado!
Teste 19: correto!
Teste 20 deu errado!
Nota exercicio 3: 1.2000000000000002
Procurando exercício 4...
Procurando exercício 5...
Procurando exercício 6...
Procurando exercício 7...
Procurando exercício 8...
Nota total: 4.2
{"scores": {"ex1": 1.0, "ex2": 2.0, "ex3": 1.2000000000000002, "ex4": 0, "ex5": 0, "ex6": 0, "ex7": 0, "ex8": 0, "ex9": 0, "ex10": 0}}
